packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -131,16 +198,85 @@ export default defineComponent({
     sharedByDisplayName: '',
     sharedTime: 0,
     sharedItem: null,
-    shareIndicators: []
+    shareIndicators: [],
+    copiedDirect: false,
+    copiedEos: false,
+    timeout: null
   }),
   computed: {
-    ...mapGetters('Files', ['versions', 'sharesTree', 'sharesTreeLoading']),
+    ...mapGetters('Files', [
+      'versions',
+      'sharesTree',
+      'sharesTreeLoading',
+      'currentFileOutgoingCollaborators'
+    ]),
     ...mapGetters(['user', 'getToken', 'configuration']),
 
+    sharedWithLabel() {
+      return this.$gettext('Shared with')
+    },
     file() {
       return this.displayedItem.value
     },
 
+    runningOnEos() {
+      return !!this.configuration?.options?.runningOnEos
+    },
+
+    hasSharees() {
```

Comment:
From here to line 279 are not necessary in this component (possibly rebase-gone-wrong?), could you double-check and delete if you agree?

Reply:
Correct, this was included by mistake. I'll remove it.